packages/config/src/json/json.config.ts

```diff
@@ -252,9 +262,60 @@ export class ConfigJson {
       bounds,
       files,
     };
+
+    output.overviews = await ConfigJson.findImageryOverviews(output);
     this.mem.put(output);
     // Ensure there is also a tile set for each imagery set
```

Comment:
These two lines comment out code seems not useful.

Reply:
I didnt want to modify this, could remove it later?